[2022-02-09 09:00:50,837] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-06-01T23:00:00+00:00 [queued]>
[2022-02-09 09:00:50,850] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-06-01T23:00:00+00:00 [queued]>
[2022-02-09 09:00:50,850] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:00:50,850] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 09:00:50,850] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:00:50,864] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2021-06-01 23:00:00+00:00
[2022-02-09 09:00:50,871] {standard_task_runner.py:52} INFO - Started process 462 to run task
[2022-02-09 09:00:50,878] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'local_to_gcs_task', 'scheduled__2021-06-01T23:00:00+00:00', '--job-id', '501', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingestee_gcs_dag.py', '--cfg-path', '/tmp/tmpx4a7ogza', '--error-file', '/tmp/tmpfwc9bqnj']
[2022-02-09 09:00:50,879] {standard_task_runner.py:77} INFO - Job 501: Subtask local_to_gcs_task
[2022-02-09 09:00:50,956] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-06-01T23:00:00+00:00 [running]> on host cc021fb126ed
[2022-02-09 09:00:51,009] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 09:00:51,036] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2021-06-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-06-01T23:00:00+00:00
[2022-02-09 09:01:05,036] {python.py:175} INFO - Done. Returned value was: None
[2022-02-09 09:01:05,051] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=local_to_gcs_task, execution_date=20210601T230000, start_date=20220209T090050, end_date=20220209T090105
[2022-02-09 09:01:05,108] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 09:01:05,163] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-09 09:16:40,173] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-06-01T23:00:00+00:00 [queued]>
[2022-02-09 09:16:40,196] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-06-01T23:00:00+00:00 [queued]>
[2022-02-09 09:16:40,197] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:16:40,197] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 09:16:40,197] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:16:40,217] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2021-06-01 23:00:00+00:00
[2022-02-09 09:16:40,225] {standard_task_runner.py:52} INFO - Started process 624 to run task
[2022-02-09 09:16:40,232] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'local_to_gcs_task', 'scheduled__2021-06-01T23:00:00+00:00', '--job-id', '605', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingesteee_gcs_dag.py', '--cfg-path', '/tmp/tmpxrpwkv50', '--error-file', '/tmp/tmpvox_q_lk']
[2022-02-09 09:16:40,233] {standard_task_runner.py:77} INFO - Job 605: Subtask local_to_gcs_task
[2022-02-09 09:16:40,314] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-06-01T23:00:00+00:00 [running]> on host 75953ff4f3d5
[2022-02-09 09:16:40,370] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 09:16:40,394] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2021-06-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-06-01T23:00:00+00:00
[2022-02-09 09:16:53,572] {python.py:175} INFO - Done. Returned value was: None
[2022-02-09 09:16:53,588] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=local_to_gcs_task, execution_date=20210601T230000, start_date=20220209T091640, end_date=20220209T091653
[2022-02-09 09:16:53,648] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 09:16:53,683] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
